library/src/main/java/com/getstream/sdk/chat/enums/MessageStatus.java

```diff
@@ -3,7 +3,8 @@
 public enum MessageStatus {
     RECEIVED("received"),
     FAILED("failed"),
-    SENDING("sending");
+    SENDING("sending"),
+    PENDING("pending");
```

Comment:
@tschellenbach does this make sense for you now that have syncStatus as well?

Reply:
I have removed `MessageStatus` and `MessageStatusConverter`, We can handle with `Sync` for `MessageStatus` behavior.